CONTRIBUTING.md

```diff
@@ -1,6 +1,132 @@
-Contributions from everyone, experienced and inexperienced, are welcome!
-If you don't know where to start, look at the open [Issues](https://github.com/StingraySoftware/stingray/issues) and/or get involved in our [Slack channel ](http://slack-invite.timelabtechnologies.com/).
-Please follow the [Astropy contribution guidelines](http://docs.astropy.org/en/stable/development/workflow/development_workflow.html), 
-and the [Astropy coding guidelines](http://docs.astropy.org/en/stable/development/codeguide.html#coding-style-conventions). 
-This code is written in Python 3.4+, and we strive to be compatible with Python 2.7. 
-Tests run at each commit during Pull Requests, so it is easy to single out points in the code that break this compatibility.
+# Contributing to stingray
+
+---
+
+> All great things have small beginnings.
+
+Hello there! We love and appreciate every small contribution you can make to improve stingray!
+We are proudly open source and believe our(yes! yours as well) work will help enhance the quality of research around the world.  We want to make contributing to stingray as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+A successful project is not just built by amazing programmers but by an amalgamation of coders, testers,  reviewers and even documentation writers. There are a few guidelines that we need all contributors to follow so that we can have a chance of keeping on top of things.
```

Comment:
"amalgamation" looks out of context?

Reply:
Changed to "by the combined, unrelenting efforts of"